aws/client.go

```diff
@@ -92,6 +96,13 @@ func (t *ResilientTransport) tries(req *http.Request) (res *http.Response, err e
 	return
 }
 
+func RandExpBackoff(try int) {
+	r := rand.New(rand.NewSource(time.Now().UnixNano()))
+	time.Sleep(time.Duration(r.Intn(1000)) *
+		time.Millisecond *
+		time.Duration(math.Exp2(float64(try))))
+}
+
```

Comment:
Does this really work better than just changing the 100 in `ExpBackoff` to 500?  (Along with upping MaxTries to 10.)